.ci/build.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash -l
+
+if [ -z "$DEBUG" ]
+then
+    set -x
+fi
+
+echo "DEBUG: WORKSPACE = ${WORKSPACE}"
+
+module load dev/cuda10.0
+module load hpcx-gcc
```

Comment:
there are many scripts in ```.ci/``` folder (good).
all of them doing similar logic as lines 1-11
why not to have settings.sh script and source it from all scripts in .ci?
Once we switch to cuda11 - you will need to change settings.sh one file only and not all scripts.
also set -o pipeline here would save it from pipeline.

Reply:
Fixed.